### PR TITLE
Adding HTTP test Jenkins stage.

### PR DESCRIPTION
**Pull request by jmencak**
@chaitanyaenr PTAL.

### CODE REVIEW

File: pipeline-scripts/http_test.groovy [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+			def clear_results = http_test_properties['CLEAR_RESULTS']
+			def move_results = http_test_properties['MOVE_RESULTS']
+			def containerized = http_test_properties['CONTAINERIZED']
+			def gun = http_test_properties['GUN']
+			def load_generator_nodes = http_test_properties['LOAD_GENERATOR_NODES']
+			def cl_projects = http_test_properties['CL_PROJECTS']
+			def cl_templates = http_test_properties['CL_TEMPLATES']
+
+                        // debug info
+                        println "TEST_CFG: '${test_cfg}'"
```
Comment (chaitanyaenr) [inline pipeline-scripts/http_test.groovy]: Wrong spacing.

**Review by chaitanyaenr — CHANGES_REQUESTED** [review]
Good to go once the spacing issues are addressed.

File: pipeline-scripts/http_test.groovy [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                                                [$class: 'BooleanParameterValue', name: 'CLEAR_RESULTS', value: Boolean.valueOf(clear_results) ],
+                                                [$class: 'BooleanParameterValue', name: 'MOVE_RESULTS', value: Boolean.valueOf(move_results) ],
+                                                [$class: 'BooleanParameterValue', name: 'CONTAINERIZED', value: Boolean.valueOf(containerized) ],
+                                                [$class: 'StringParameterValue', name: 'GUN', value: gun ],
+                                                [$class: 'StringParameterValue', name: 'LOAD_GENERATOR_NODES', value: load_generator_nodes ],
+                                                [$class: 'StringParameterValue', name: 'CL_PROJECTS', value: cl_projects ],
+                                                [$class: 'StringParameterValue', name: 'CL_TEMPLATES', value: cl_templates ],
+                                            ]
+                        } catch (Exception e) {
+                       	    echo "HTTP scale-test Job failed with the following error: "
```
Comment (chaitanyaenr) [inline pipeline-scripts/http_test.groovy]: ditto.

File: pipeline-scripts/http_test.groovy [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                        // Run http_test job
+                        try {
+                            http_test_build = build job: 'http-scale-test',
+                                parameters: [   [$class: 'LabelParameterValue', name: 'node', label: node_label ],
+                                                [$class: 'StringParameterValue', name: 'TEST_CFG', value: test_cfg ],
+                                                [$class: 'StringParameterValue', name: 'JUMP_HOST', value: jump_host ],
+                                                [$class: 'StringParameterValue', name: 'JUMP_HOST_USER', value: jump_host_user ],
+						[$class: 'BooleanParameterValue', name: 'USE_PROXY', value: Boolean.valueOf(use_proxy) ],
+                                                [$class: 'StringParameterValue', name: 'PROXY_HOST', value: proxy_host ],
+                                                [$class: 'StringParameterValue', name: 'PROXY_USER', value: proxy_user ],
```
Comment (chaitanyaenr) [inline pipeline-scripts/http_test.groovy]: ditto.

File: pipeline-scripts/http_test.groovy [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+			println "CL_TEMPLATES: '${cl_templates}'"
+
+                        // Run http_test job
+                        try {
+                            http_test_build = build job: 'http-scale-test',
+                                parameters: [   [$class: 'LabelParameterValue', name: 'node', label: node_label ],
+                                                [$class: 'StringParameterValue', name: 'TEST_CFG', value: test_cfg ],
+                                                [$class: 'StringParameterValue', name: 'JUMP_HOST', value: jump_host ],
+                                                [$class: 'StringParameterValue', name: 'JUMP_HOST_USER', value: jump_host_user ],
+						[$class: 'BooleanParameterValue', name: 'USE_PROXY', value: Boolean.valueOf(use_proxy) ],
```
Comment (chaitanyaenr) [inline pipeline-scripts/http_test.groovy]: ditto.

**Review by chaitanyaenr — CHANGES_REQUESTED** [review]

**Comment by jmencak** [comment]
reformatted, space2tabs @chaitanyaenr , ptal

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Comment by chaitanyaenr** [comment]
Thanks for the PR @jmencak.

**Pull request merged**